src/crypto/crypto_util.cc

```diff
@@ -105,6 +103,25 @@ int NoPasswordCallback(char* buf, int size, int rwflag, void* u) {
   return 0;
 }
 
+bool ProcessFipsOptions() {
+  /* Override FIPS settings in cnf file, if needed. */
```

Comment:
I think `config` or `configuration` is more readable than `cfg`:
```suggestion
  /* Override FIPS settings in configuration file, if needed. */
```
Edit: I see you didn't write this, but it might as well be fixed now.

Reply:
I've updated this now, thanks